watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -54,61 +55,61 @@ def test_solve(self, system_frame):
         )
         assert value(m.fs.Treated.properties[0].conc_mass_comp["S_F"]) == pytest.approx(
-            0.00022875, rel=1e-3
+            0.00027610, rel=1e-3
         )
         assert value(m.fs.Treated.properties[0].conc_mass_comp["S_I"]) == pytest.approx(
             0.057450, rel=1e-3
         )
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_N2"]
-        ) == pytest.approx(0.024905, rel=1e-3)
+        ) == pytest.approx(0.052060, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_NH4"]
-        ) == pytest.approx(0.031034, rel=1e-3)
+        ) == pytest.approx(0.00017686, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_NO3"]
-        ) == pytest.approx(0, abs=1e-6)
+        ) == pytest.approx(0.0060483, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_O2"]
-        ) == pytest.approx(0.0077457, rel=1e-3)
+        ) == pytest.approx(0.0076767, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_PO4"]
-        ) == pytest.approx(0.7651583, rel=1e-3)
+        ) == pytest.approx(0.65267, rel=1e-3)
         assert value(m.fs.Treated.properties[0].conc_mass_comp["S_K"]) == pytest.approx(
-            0.37011, rel=1e-2
+            0.36810, rel=1e-3
         )
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_Mg"]
-        ) == pytest.approx(0.0182625, rel=1e-3)
+        ) == pytest.approx(0.018654, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["S_IC"]
-        ) == pytest.approx(0.149959, rel=1e-3)
+        ) == pytest.approx(0.15148, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["X_AUT"]
-        ) == pytest.approx(0, abs=1e-6)
+        ) == pytest.approx(0.00041597, rel=1e-3)
         assert value(m.fs.Treated.properties[0].conc_mass_comp["X_H"]) == pytest.approx(
-            0.0125387, rel=1e-3
+            0.013371, rel=1e-3
         )
         assert value(m.fs.Treated.properties[0].conc_mass_comp["X_I"]) == pytest.approx(
-            0.0119245, rel=1e-3
+            0.012360, rel=1e-3
         )
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["X_PAO"]
-        ) == pytest.approx(0.0132445, rel=1e-3)
+        ) == pytest.approx(0.011077, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["X_PHA"]
-        ) == pytest.approx(6.445095e-06, rel=1e-3)
+        ) == pytest.approx(5.0389e-06, rel=1e-3)
         assert value(
             m.fs.Treated.properties[0].conc_mass_comp["X_PP"]
-        ) == pytest.approx(0.00441488, rel=1e-3)
+        ) == pytest.approx(0.0036998, rel=1e-3)
         assert value(m.fs.Treated.properties[0].conc_mass_comp["X_S"]) == pytest.approx(
-            0.00021346, rel=1e-3
+            0.00021570, rel=1e-3
```

Comment:
I suppose my uncertainty has to do with the concentrations of `X` components that we should expect to see in treated effluent (at least in terms of order of magnitude).